tools/bin/ci_credentials.sh

```diff
@@ -236,8 +236,6 @@ read_secrets source-klaviyo "$KLAVIYO_TEST_CREDS"
 read_secrets source-lemlist "$SOURCE_LEMLIST_TEST_CREDS"
 read_secrets source-lever-hiring "$LEVER_HIRING_INTEGRATION_TEST_CREDS"
 read_secrets source-looker "$LOOKER_INTEGRATION_TEST_CREDS"
-read_secrets source-linkedin-ads "$SOURCE_LINKEDIN_ADS_TEST_CREDS"
```

Comment:
why are these removed?

Reply:
Now all secrets should be saved into GSM storage. Thus we are trying to remove all secrets from GitHub gradually